common/app/experiments/Experiments.scala

```diff
@@ -82,3 +83,13 @@ object BillboardsInMerchSlots
       sellByDate = LocalDate.of(2023, 2, 1),
       participationGroup = Perc0E,
     )
+
+object ServerSideLiveblogInlineAds
+    extends Experiment(
+      name = "server-side-liveblog-inline-ads",
+      description =
+        "Test whether we can load liveblog inline ads server-side without negative effects on user experience or revenue",
+      owners = Seq(Owner.withGithub("@guardian/commercial-dev")),
+      sellByDate = LocalDate.of(2023, 2, 1),
+      participationGroup = Perc0E,
```

Comment:
This shares a participation group, as the Fastly buckets only go up to "E". However I think this is fine for the following reasons:
 - This is a 0% test and won't be seen by users that don't opt-in
 - The other test in this participation group is related to fronts, not liveblog pages

Reply:
Although there is a unit test that is failing because of the shared participation groups.

Reply:
I think it's because only one of the `Perc0C` tests is in the set of active experiments.

We could maybe remove the merchandising slot billboard from the list of active experiments, since we won't pick it up this quarter?

Reply:
Thanks Chris